code/site/components/com_pages/view/collection/tmpl/newsfeed.rss.php

```diff
@@ -11,17 +11,18 @@
 
 <rss version="2.0"
      xmlns:atom="http://www.w3.org/2005/Atom"
-     xmlns:sy="http://purl.org/rss/1.0/modules/syndication/">
+     xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
+     xmlns:media="http://search.yahoo.com/mrss/">
 
     <channel>
-        <title><?= translate('Pages') ?> - <?= escape($sitename)?></title>
-        <description><![CDATA[<?= $description ?>]]></description>
-        <link><?= url() ?></link>
-        <? if (!empty($image)): ?>
+        <title><?= page()->title ?></title>
+        <description><?= page()->summary ?></description>
+        <link><?= route(page()->path.'/'.page()->slug) ?></link>
+        <? if (!empty($page->image)): ?>
             <image>
-                <url><?=$image?></url>
-                <title><?= translate('Pages') ?> - <?= escape($sitename)?></title>
-                <link><?= route('format=html') ?></link>
+                <url>baseurl://image://<?= $page->image ?></url>
```

Comment:
@johanjanssens This leads to double slash as baseurl:// ends with one, and image:// starts with another. Like this: 
`<media:content medium="image" url="http://joomlatools.test//images/blog/2018/linker-seo/cover.png"/>`

Reply:
@ercanozkaya I have removed the use of image:// in the newsfeeds template and refactored all blog images to include images/ this provided consistency.